.github/workflows/monorepo-validate.yaml

```diff
@@ -62,6 +62,9 @@ jobs:
     runs-on: ${{ fromJSON(vars.RUNNER) }}
     timeout-minutes: 15
     steps:
+      - uses: browser-actions/setup-chrome@c785b87e244131f27c9f19c1a33e2ead956ab7ce # v1.7.3
+        with:
+          install-dependencies: true
```

Comment:
Technically I don't think we need this as it looks like they [include dependencies](https://github.com/hashicorp/crt-github-runners/blob/16d07c4353b8e1c2e87e9843e5696149c96d399e/packer/scripts/installers/chrome-dep.sh) for us but it doesn't hurt I imagine